modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/subscriptions/SubscriptionDeleteTestCase.java

```diff
@@ -0,0 +1,412 @@
+
+        //verify all subscribers got at least one message
+        boolean allSubscribersGotMessages = false;
+        if(consumerClient1.getReceivedMessageCount() > 0
+                && consumerClient2.getReceivedMessageCount() > 0
+                && consumerClient3.getReceivedMessageCount() > 0) {
+            allSubscribersGotMessages = true;
+        }
+
+        Assert.assertEquals(allSubscribersGotMessages, true, "All queue subscribers are not receiving messages");
+
+        QueueSubscriptionsPage queueSubscriptionsPage = homePage.getQueueSubscriptionsPage();
+        boolean result = queueSubscriptionsPage.closeTopSubscription();
+
+        Assert.assertEquals(result, true, "Closing queue subscriber was not successful");
+
+        //verify one of the subscribers now does not get new messages
+
+        long messageCountSub1 = consumerClient1.getReceivedMessageCount();
+        long messageCountSub2 = consumerClient2.getReceivedMessageCount();
+        long messageCountSub3 = consumerClient3.getReceivedMessageCount();
+
+        try {
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            //ignore
+        }
+
+        long messageCountNewSub1 = consumerClient1.getReceivedMessageCount();
+        long messageCountNewSub2 = consumerClient2.getReceivedMessageCount();
+        long messageCountNewSub3 = consumerClient3.getReceivedMessageCount();
+
+        boolean oneReceiverIsStopped = false;
+
+        if(messageCountNewSub1 == messageCountSub1) {
+            oneReceiverIsStopped = true;
+        } else if(messageCountNewSub2 == messageCountSub2) {
+            oneReceiverIsStopped = true;
+        } else if(messageCountNewSub3 == messageCountSub3) {
+            oneReceiverIsStopped = true;
+        }
+
+        Assert.assertEquals(oneReceiverIsStopped, true, "None of the queue subscribers stopped even if"
+                + "subscriber is stopped via UI ");
+
+        //close all subscriptions
+        consumerClient1.stopClient();
+        consumerClient2.stopClient();
+        consumerClient3.stopClient();
+    }
+
+
+    /**
+     * 1. Create 3 topic subscribers for topic "subDeleteTopic"
+     * 2. Log into UI and close one subscriptions
+     * 3. Verify subscription close is successful and only 2 other subscription entries are shown
+     * 4. Verify one of subscribers does not get new messages
+     * @throws IOException
+     * @throws XPathExpressionException
+     * @throws JMSException
+     * @throws NamingException
+     * @throws AndesClientException
+     */
+    @Test(groups = {"wso2.mb", "topic"})
+    public void performTopicSubscriptionCloseTestCase() throws IOException, XPathExpressionException,
+            JMSException, NamingException, AndesClientException, AndesClientConfigurationException,
+            CloneNotSupportedException {
+
+        String topicName = "subDeleteTopic";
+        long expectedMessageCount = 1000L;
+        long messageCountToSend = 1000L;
```

Comment:
The three tests are individual tests. No value addition making it is a constant. There can be chances we may need to update one test.